modelservice/worker_test.py

```diff
@@ -0,0 +1,78 @@
+import tensorflow as tf
+import os
+import pandas as pd
+from pprint import pprint
+
+import logging
+from logging import info, error
+from sklearn.preprocessing import MinMaxScaler
+
+
+# Strictly for testing
+def main():
```

Comment:
Not at all critical, but would it be simpler to have this test call the BuildModels method of worker.py so that we always have the most recent changes from there?

Reply:
No because of the way Python packages work, if you try to `from modelservice import ...` and then run this via `python3.8 worker_test.py` it won't know what the `modelservice` package is. In order to be able to use absolute package imports like that, you have to run it from project root like `python3.8 -m modelservice <args>` and it automatically executes `__main__.py`, so there's not a good way to execute this specific `main()` function in this test file